packages/create-bison-app/template/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  postgres:
+    container_name: postgres
+    ports:
+      - 5432:5432
+    image: postgres:14
+    environment:
+      POSTGRES_USER: dev
+      POSTGRES_PASSWORD: dev
+      POSTGRES_DB: dev
+    restart: always
+    volumes:
+      - ./.data/postgres:/var/lib/postgresql/data
```

Comment:
I like the `.data/postgres` folder idea; that it leaves room for other data folders 👏